R/utils.R

```diff
@@ -270,3 +270,34 @@ list_call <- function(fn, x) {
 package_name <- function(env) {
   utils::packageName(env)
 }
+
+now_utc <- function() {
+  as.POSIXct(Sys.time(), tz = "UTC")
+}
+
+## Modified version of difftime to show timings
+## in ms and to a precision
+format_difftime <- function(time1, time2) {
+  z <- unclass(time1) - unclass(time2)
+  attr(z, "tzone") <- NULL
+  units <- if (all(is.na(z))) {
```

Comment:
why was the all() here, or min(abs(z))? I don't see when this is used with other than scalar args?

Reply:
just copied from `difftime` impl but yeah I think you're right it is not needed here